src/org/opendatakit/briefcase/ui/CharsetConverterDialog.java

```diff
@@ -285,13 +281,7 @@ protected void initialize() {
     if (defaultListModel.size() > 0) {
       listCharset.setSelectedIndex(0);
 
-      listCharset.addListSelectionListener(new ListSelectionListener() {
-
-        @Override
-        public void valueChanged(ListSelectionEvent e) {
-          updatePreview();
-        }
-      });
+      listCharset.addListSelectionListener(e -> updatePreview());
```

Comment:
When a lambda arg is not used we're trying to use `__` (two underscores) as the argument's name to express that it's ignored. This should be:
```java
listCharset.addListSelectionListener(__ -> updatePreview());
```